packages/runtime/test/transport/relationships.test.ts

```diff
@@ -492,6 +497,17 @@ describe("RelationshipTermination", () => {
         expect(result).toBeAnError(/.*/, "error.transport.messages.missingOrInactiveRelationship");
     });
 
+    // eslint-disable-next-line jest/no-disabled-tests
+    test.skip("sending Notification results in an error redirected from the Backbone", async () => {
+        const id = await ConsumptionIds.notification.generate();
+        const notificationToSend = Notification.from({ id, items: [TestNotificationItem.from({})] });
+        const result = await services1.transport.messages.sendMessage({ recipients: [services2.address], content: notificationToSend.toJSON() });
+        expect(result).toBeAnError(
+            `Cannot send message to ${services1.address.toString()} because the relationship to it is not active. In order to be able to send messages again, you have to reactivate the relationship.`,
```

Comment:
I am wondering why this test does not throw an error, because `${services2.address.toString()}` instead of `${services1.address.toString()}` should be used (`services2` is the recipient instead of `services1`).

Reply:
Yes I have noticed this also, I think there the Backbone mixed something up.

Reply:
When we came to the same conclusion, we should report this to the Backbone.

Reply:
I had a look on it in the Backbone and the reason is because they only look for the relationship between to peers and there you have 'from' and 'to' depending who created the relationship and then they put 'to' in the errorMessage so it has nothing to do with the peer who wants to send the message.

Reply:
> I had a look on it in the Backbone and the reason is because they only look for the relationship between to peers and there you have 'from' and 'to' depending who created the relationship and then they put 'to' in the errorMessage so it has nothing to do with the peer who wants to send the message.

Whoops. :) That's completely right. 🙈 I will fix it. 

@Milena-Czierlinski Regarding your suggested error message: that doesn't really work out, because when a message has multiple recipients, there are multiple relationships. So you wouldn't know which is the one that causes the problem.
I would simply use the correct address in the error message. Or is there a problem with that?

And lastly: I don't recommend you to check for error messages. At least not in their entirety. If you want to assert that the Backbone complains for the correct reason (=> the correct recipient), then checking for the string to contain the address of the recipient should be sufficient.
The problem with checking for the whole message is that this makes the test brittle. Once the backbone changes the error messages, your test will fail for no real reason.

Reply:
> I would simply use the correct address in the error message. Or is there a problem with that?

Perfectly fine, there is no problem with that. The idea of the suggestion was to reduce work when fixing the error message, but if we could get the address it would be great.

> And lastly: I don't recommend you to check for error messages. At least not in their entirety. If you want to assert that the Backbone complains for the correct reason (=> the correct recipient), then checking for the string to contain the address of the recipient should be sufficient.
> The problem with checking for the whole message is that this makes the test brittle. Once the backbone changes the error messages, your test will fail for no real reason.

Good point!

Reply:
Version 6.13.2 contains the fix for this.

Reply:
It was clear that this test will fail when you won't block Notifications anymore, that's why it was test.skip and only for debugging, but in general you are right!